examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py

```diff
@@ -223,20 +223,20 @@
 # In this part, we will use LTT to control precision.
 # At the opposite of the 2 previous method, LTT can handle non-monotonous loss.
 # The procedure consist in multiple hypothesis testing. This is why the output
-# of this procedure isn't reduce to one value of :math:`\lambda`.
+# of this procedure isn't reduce to one value of ``λ``.
 #
-# More precisely, we look after all the :math:`\lambda` that sastisfy the
+# More precisely, we look after all the ``λ`` that sastisfy the
 # following:
-# :math:`\mathbb{P}(R(\mathcal{T}_{\lambda}) \leq \alpha ) \geq 1 - \delta`,
-# where :math:`R(\mathcal{T}_{\lambda})` is the risk we want to control and
-# each :math:`\lambda`` should satisfy FWER control.
```

Comment:
That one is broken. You can use this tool (an online reStructured editor) to translate : https://feat.dlup.link/rsted

Reply:
Done!